src/data/default.json

```diff
@@ -339,6 +339,7 @@
     "settingsDialogVisible": false,
     "editDialog": false,
     "settingPath": null,
+    "previousIcon": "",
```

Comment:
this should under settings, please check the above comment for more details.

Reply:
Done